src/server/api/routes/users/v2/wallet.js

```diff
@@ -0,0 +1,33 @@
+const { Router } = require('express');
+const { walletUserCollection } = require('../../../../modules/firebase');
+const { GET_WALLET_API_SECRET } = require('../../../../config/config');
+
+const router = Router();
+
+router.get('/wallet', async (req, res, next) => {
+  try {
+    const { email, key } = req.query;
```

Comment:
key should not be in query string, use HTTP header like `Authorization` or `X-LIKERLAND-API-KEY`